gamerun.cc

```diff
@@ -0,0 +1,47 @@
+#include "gamerun.h"
+using namespace std;
+
+/*
+TO DO:
+- Add pickMove() method to Human
+- Create Levelx subclasses for Computer, each with a pickMove() method
+*/
+
+int gameRun(Observer *white, Observer *black, ChessGame *gameBoard) {
+    int turn = gameBoard->getplayerTurn();
```

Comment:
I think `chessGame.h` has the function called `getPlayerTurn()`, not `getplayerTurn()`. Just need to capitalize the "P" in player.